spyderlib/widgets/externalshell/namespacebrowser.py

```diff
@@ -437,8 +437,7 @@ def collapse(self):
         """Collapse"""
         self.sig_collapse.emit()
 
-    @Slot(list)

```

Comment:
It was producing an error... when clicking on the import button... it must have been sleeping for a while